templates/static_pages/help/accessibility-statement.html.tx

```diff
@@ -139,7 +139,7 @@
     <br>
     <p><a class="govuk-link" href="https://companieshouse.blog.gov.uk/category/user-research/">Read our blogs about usability</a>
           to find out more about our user research work.</p>
-    <p>This statement was prepared on 7 May 2020. It was last updated on 11 October 2024.</p>
+    <p>This statement was prepared on 7 May 2020. It was last updated on the 8<sup>th</sup> of Jan 2025.</p>
```

Comment:
This date should the the 22nd of Jan, as that's when we're planning to release it. Also I think maybe stick with the previous format, so 22 January 2025